rootfs/scheduler/__init__.py

```diff
@@ -142,7 +142,7 @@
                 "value":"$id"
             },
             {
-                "name":"DEIS_RELEASE",
+                "name":"WORKFLOW_RELEASE",
```

Comment:
Does anyone know why we inject this into the RCs? It uses `$appversion` and well... that's the application release version - Can we remove it?

Either way, I don't think this should be renamed if it is not being used for release purposes